app.js

```diff
@@ -0,0 +1,59 @@
+const text = document.getElementById("text");
```

Comment:
Sayfa yüklendiğinde #FFFFFF rengi  karşımıza çıkıyor. Fakat bu rengi herhangi bir colorBox'a atayamıyoruz. Bunu çözmek için document'e DOMContentLoaded event ekleyebilirsin. Böylelikle HTML kodu parse edilip, DOM Tree oluşturulduğu gibi randomColorCode fonksiyonu ile random bir renk oluşturup, onu text kısmına yazarak ve background'a vererek kullanabilirsin.